src/Wallabag/CoreBundle/Resources/translations/messages.fr.yml

```diff
@@ -1,422 +1,422 @@
+            re_fetch_content: "Recharger le contenu"
+            delete: "Supprimer"
+            add_a_tag: "Ajouter un tag"
+            share_content: "Partager"
+            share_email_label: "Courriel"
+            public_link: "Lien public"
+            delete_public_link: "Supprimer lien public"
+            download: "Télécharger"
+            print: "Imprimer"
             problem:
-                label: 'Un problème ?'
-                description: "Est-ce que cet article s'affiche mal ?"
-        edit_title: 'Modifier le titre'
-        original_article: 'original'
-        annotations_on_the_entry: '{0} Aucune annotation|{1} Une annotation|]1,Inf[ %count% annotations'
-        created_at: 'Date de création'
+                label: "Un problème ?"
+                description: "Est-ce que cet article s’affiche mal ?"
+        edit_title: "Modifier le titre"
+        original_article: "original"
+        annotations_on_the_entry: "{0} Aucune annotation|{1} Une annotation|]1,Inf[ %count% annotations"
+        created_at: "Date de création"
     new:
-        page_title: 'Sauvegarder un nouvel article'
-        placeholder: 'http://website.com'
+        page_title: "Sauvegarder un nouvel article"
+        placeholder: "http://website.com"
         form_new:
-            url_label: Url
+            url_label: "Adresse"
     edit:
-        page_title: 'Éditer un article'
-        title_label: 'Titre'
-        url_label: 'Url'
-        is_public_label: 'Public'
-        save_label: 'Enregistrer'
+        page_title: "Éditer un article"
+        title_label: "Titre"
+        url_label: "Adresse"
+        is_public_label: "Public"
+        save_label: "Enregistrer"
     public:
-        shared_by_wallabag: "Cet article a été partagé par <a href='%wallabag_instance%'>wallabag</a>"
+        shared_by_wallabag: "Cet article a été partagé par <a href=’%wallabag_instance%’>wallabag</a>"
 
 about:
-    page_title: 'À propos'
+    page_title: "À propos"
     top_menu:
-        who_behind_wallabag: "L'équipe derrière wallabag"
-        getting_help: "Besoin d'aide"
-        helping: 'Aider wallabag'
-        contributors: 'Contributeurs'
-        third_party: 'Librairies tierces'
+        who_behind_wallabag: "L’équipe derrière wallabag"
+        getting_help: "Besoin d’aide"
+        helping: "Aider wallabag"
+        contributors: "Contributeurs"
+        third_party: "Librairies tierces"
     who_behind_wallabag:
-        developped_by: 'Développé par'
-        website: 'Site web'
-        many_contributors: 'Et plein de contributeurs ♥ <a href="https://github.com/wallabag/wallabag/graphs/contributors">sur Github</a>'
-        project_website: 'Site web du projet'
-        license: 'Licence'
-        version: 'Version'
+        developped_by: "Développé par"
+        website: "Site web"
+        many_contributors: "Et plein de contributeurs ♥ <a href="https://github.com/wallabag/wallabag/graphs/contributors">sur Github</a>"
```

Comment:
Nope, keep `'` around that line otherwise the yaml parser will fail because the line also contains `"`.
And that's the same for all other lines which contains `"`.
Or convert `"` to `\"`